src/containers/HookManager/HookManager.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import HookLoader from '../../components/HookLoader/HookLoader';
+import HookList from '../../components/HookList/HookList';
+import HookEditor from '../../components/HookEditor/HookEditor';
+import classes from './HookManager.module.css';
+
+class HookManager extends Component {
+  state = {
+    hooks: [],
+    activeHookId: "",
+    loaderInput: "",
+    loadStatus: ""
+  }
+
+  handleSubmitURL = (event) => {
+    event.preventDefault();
+    const input = this.state.loaderInput;
+    if (input === "") { // this should be a valid URL check
+      this.setState({ loadStatus: "invalid" });
+      return;
+    }
+    this.setState({ loadStatus: "loading" });
+    axios.get(input)
```

Comment:
network request should be handled in actions or sagas. Move this when you setup reducers